Generals/Code/GameEngine/Source/GameNetwork/GameSpy/Thread/GameResultsThread.cpp

```diff
@@ -33,6 +33,7 @@
 #include "GameNetwork/GameSpy/GameResultsThread.h"
 #include "mutex.h"
 #include "thread.h"
+#include <eh.h>
```

Comment:
Is this meant to be here?

In multiple GameSpy files.

Reply:
excluded